src/initializers/kafka/kafka.ts

```diff
@@ -25,8 +25,21 @@ export default class Kafka {
     logger.info(`Kafka connected ${brokers.join(',')}`);
   }
 
-  public async send(topic: string, message: string | Buffer, key?: string, partition?: number) {
-    const { key: _key, partition: _partition, offset } = await this.producer.send(topic, message, partition, key);
+  public async send(
+    topic: string,
+    message: string | Buffer,
+    key?: string,
```

Comment:
We could add key, partition, headers as a third options argument.
This is backwards incompatible but it will be more readable in the future. 
We could add a couple of lines to make it backwards compatible if we need to eg.
```
if options is string 
   then key=options
```

Reply:
Let's do this in another PR